### DISCUSSION TITLE
Error: spawn free EACCES

### DISCUSSION THREAD

**Discussion by xyzzur** [Heroic-Games-Launcher/HeroicGamesLauncher#3182] [Q&A]
Good night! I need help, I installed the application, but when I try to log in, it shows an error, maybe someone has already solved this problem. Thank you!
![Знімок екрана з 2023-11-01 23-24-02](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/104593322/b0604a0a-d173-4d52-9762-c61cce9da70c)

**Comment by arielj** [comment]
Update to heroic 2.10.0, the snap works a bit better

if you still have issues, use appimage, flatpak or .deb, the snap is not fully functional
